### PR TITLE
Stop debug APK builds when there are no relevant changes

### LINKED ISSUE #2264: Stop the debug apk build when there are no changes
### About this discussion

It takes several minutes to build the debug apk, but if Lawnicons doesn't change, then the build action is triggered in vain.

I suggest adding a check to the build action so that it is successfully canceled when there are no updates. This would help in some cases to merge the PR without waiting for the build.

### PR DESCRIPTION
**Pull request by Hamster45105**
## Description

This PR ensures that the `Build Debug APK` action will not run on commits/pull requests that don't affect the app itself.

I also added a localisation section to the nightly release notes. All PRs should be included in the release notes, except for things with just the `docs` label or no label.

Fixes #2264 

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: General change (non-breaking change that doesn't fit the above categories, such as copyediting)

### CODE REVIEW

File: .github/workflows/build_debug_apk.yml [hunk 1]
```diff
@@ -5,7 +5,29 @@ on:
...
       - develop
+    paths-ignore:
+      - '**.md'
+      - 'LICENSE'
+      - '.idea/**'
+      - 'docs/**'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.gitignore'
+      - '.gitattributes'
+      - '.github/workflows/**'
```
Comment (Goooler) [inline .github/workflows/build_debug_apk.yml:16]:
```suggestion
```

File: .github/workflows/build_debug_apk.yml [hunk 2]
```diff
@@ -5,7 +5,29 @@ on:
...
   pull_request:
+    paths-ignore:
+      - '**.md'
+      - 'LICENSE'
+      - '.idea/**'
+      - 'docs/**'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.gitignore'
+      - '.gitattributes'
+      - '.github/workflows/**'
```
Comment (Goooler) [inline .github/workflows/build_debug_apk.yml:28]:
```suggestion
```

**Review by Goooler — APPROVED** [review]

File: .github/release.yml [hunk 3]
```diff
@@ -12,3 +12,6 @@ changelog:
     - title: 🧑‍💻 Dependencies
       labels:
         - dependencies
+    - title: 💬 Localisation
```
Comment (x9136) [inline .github/release.yml:15]: Due to the fact that the IT world largely comes from America, it is more common to see "localization". Although your version is also correct, but for British English.

Comment (Hamster45105) [reply .github/release.yml]: Considering things like the contributing guidelines are written using American English, for the sake of consistency, I have updated the spelling.

**Review by x9136 — APPROVED** [review]

**Review by SuperDragonXD — APPROVED** [review]

**Pull request merged**